src/test/java/net/datafaker/integration/FakerIntegrationTest.java

```diff
@@ -157,6 +150,8 @@ void testExceptionsNotCoveredInAboveTest(Locale locale, Random random) {
     private static Stream<Arguments> dataParameters() {
         List<Arguments> arguments = new ArrayList<>();
         arguments.add(Arguments.of(Locale.ENGLISH, new Random()));
+        arguments.add(Arguments.of(new Locale("en-US"), new Random()));
```

Comment:
we really should stop using this `-` or `_` approach, it's not something we support officially, and it's removed from all documentation AFAIK.

Reply:
Do you mean replacing `new Locale("en-US")` by `new Locale("en", "US")`?
Yes, why not. 

P.S. Starting from Java 19, this constructor is also deprecated. Should be replaced by `Locale.of("en", "US")`.

Reply:
Yes, the `new Locale("en-US")` is not an officially supported Locale, that's something Javafaker introduced, and ideally, it should be removed from the code, not added.